packages/frontend/src/pages/Movies.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import axios from 'axios'
+import configAxios from '../utils/API'
+
+export default class Movies extends React.Component {
+
+    state = {
+        persons: []
+    }
+
+    componentDidMount() {
+
+        const persons1: any[] = []
+        axios.get(`https://jsonplaceholder.typicode.com/users`)
+            .then(res => {
```

Comment:
Przenieś proszę cały axios do osobnego pliku i tu tylko go wywołuj, tak eby logika była w innym pliku a tu tylko widok.

Reply:
scieszka do env. i axios skonfigurowac np. baseurl

Reply:
> scieszka do env. i axios skonfigurowac np. baseurl

hmm zmieniłam na: 
`axiosConfig.get(axiosConfig.defaults.baseURL + `/users`)`
dodałam config instance:
`import axios from 'axios'

const instance = axios.create({
    baseURL: 'https://jsonplaceholder.typicode.com',
    timeout: 1000,
});

export default instance`

Reply:
Jest juz bardzo fajnie, tylko jeszcze chodziło mi o to, ze tą część gdzie masz url, czyli 
  axiosConfig.get(axiosConfig.defaults.baseURL + `/users`) mozna by do osobnego pliku wrzucić, tak zeby wszystkie url do pobierania filmów, czy tez do pojedynczego filmu mieć w jednym pliku zeby później móc to łatwo znaleść, a tu wywoływać funkcje i dalej .then(resp =>  { ...})